AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1368,7 +1371,7 @@ codedosa.com,newslibrary.naver.com,insight.co.kr,jootc.com,gamefinity.id,blog.da
 insight.co.kr,jootc.com,blog.daum.net,comic.naver.com,cafe.naver.com#%#//scriptlet('set-constant', 'document.onselectstart', 'null')
 codedosa.com,gradium.co.kr,insight.co.kr,jootc.com,gamefinity.id,blog.daum.net,comic.naver.com,cafe.naver.com#%#//scriptlet('set-constant', 'document.ondragstart', 'null')
 comic.naver.com#%#//scriptlet('remove-attr', 'onselectstart|ondragstart|oncontextmenu', 'img[id^="content_image_"]')
-egloos.com,novel.naver.com#%#//scriptlet('remove-attr', 'onselectstart|ondragstart|oncontextmenu', 'body')
+outdoornews.co.kr,egloos.com,novel.naver.com#%#//scriptlet('remove-attr', 'onselectstart|ondragstart|oncontextmenu', 'body')
```

Comment:
It seems this one is enough.

Reply:
Okay. https://github.com/AdguardTeam/AdguardFilters/pull/90512/commits/ce1c4e69e929ab00ec350ca5cdaaeac3fe7800b8